src/hearings/utils/hearings.utils.ts

```diff
@@ -256,4 +256,29 @@ export class HearingsUtils {
       HearingsUtils.modifyHearingDetailsYear(input?.hearingDetails?.hearingWindow);
     }
   }
+
+  public static doArraysDiffer(array: string[], arrayToCompare: string[]): boolean {
+    return !_.isEqual(this.standardiseStringArray(array), this.standardiseStringArray(arrayToCompare));
+  }
+
+  private static standardiseStringArray(array: string[] | null | undefined): string[] | undefined {
+    return array && array.length > 0 ? [...array].sort() : undefined;
+  }
+
+  public static haveAdditionalFacilitiesChanged(
```

Comment:
Please add a return type to this function, particularly as it's an odd one
should be :{Boolean, Boolean} by the looks of it